kubernetes/celery-beat/deployment.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: celery-beat-deployment
+  labels:
```

Comment:
[Importance 8/10]

We are missing labels/annotation used by helm to see what it manage

https://helm.sh/docs/chart_best_practices/labels/

Reply:
done